include/s3select_oper.h

```diff
@@ -516,6 +520,18 @@ class value
     type = value_En_t::FLOAT;
   }
 
+  void set_true() 
+  {
+    __val.num = 1;
+    type = value_En_t::DECIMAL;
```

Comment:
it may needs for new type (BOOLEAN)
to differentiate between 1/0 of decimal with true/false of boolean